module/zfs/arc.c

```diff
@@ -864,6 +864,16 @@ static inline void arc_hdr_clear_flags(arc_buf_hdr_t *hdr, arc_flags_t flags);
 static boolean_t l2arc_write_eligible(uint64_t, arc_buf_hdr_t *);
 static void l2arc_read_done(zio_t *);
 
+/*
+ * L2ARC TRIM
+ * l2arc_trim_ahead : A ZFS module parameter that controls how much ahead of
+ * 		the current write size we should TRIM. It is defined as a
+ * 		percentage of the write size. The minimum TRIM size is 64MB.
+ * 		If set to 100 and the write size is greater than 64MB we don't
+ * 		trim ahead. The default is 0, TRIM is disabled for L2ARC.
```

Comment:
Update to match the final text in the man page description.

Can you also include a brief explanation with the rational for why L2ARC trimming is disabled by default.